src/main/java/org/folio/consortia/controller/ErrorHandlingController.java

```diff
@@ -70,6 +71,15 @@ public Errors handleValidationErrors(Exception e) {
     return createExternalError(e.getMessage(), VALIDATION_ERROR);
   }
 
+  @ResponseStatus(HttpStatus.UNAUTHORIZED)
+  @ExceptionHandler({
+    InvalidTokenException.class
+  })
+  public Errors handleTokenErrors(Exception e) {
+    log.error("Handle validation errors", e);
```

Comment:
better to rename to rename to: "Handle token validation errors"

Reply:
fixed